idb/postgres/postgres.go

```diff
@@ -2362,13 +2419,26 @@ func (db *IndexerDb) GetAccounts(ctx context.Context, opts idb.AccountQueryOptio
 		err = fmt.Errorf("account query %#v err %v", query, err)
 		out <- idb.AccountRow{Error: err}
 		close(out)
-		tx.Rollback()
 		return out
 	}
 	go db.yieldAccountsThread(req)
 	return out
 }
 
+func (db *IndexerDb) getAccounts(opts idb.AccountQueryOptions) <-chan idb.AccountRow {
```

Comment:
We should update the old code to use the new signature instead of adding wrapper functions.